presentation/components/src/presentation-components/table/DataProvider.ts

```diff
@@ -439,6 +439,8 @@ class CellsBuilder extends PropertyRecordsBuilder {
         const expandedNestedContentField = this._sameInstanceFields[hierarchy.field.name];
         expandedNestedContentField.name = hierarchy.field.name;
         hierarchy.field = expandedNestedContentField;
+        const updatedField = createFieldHierarchies([expandedNestedContentField], true);

```

Comment:
I suggest renaming the variable to `updatedFieldHierarchy`.

Reply:
Done